src/test/scala/org/neo4j/spark/SparkConnectorScalaSuiteIT.scala

```diff
@@ -83,6 +85,8 @@ object SparkConnectorScalaSuiteIT {
 @Suite.SuiteClasses(Array(
   classOf[SchemaServiceTSE],
   classOf[DataSourceReaderTSE],
-  classOf[DataSourceWriterTSE]
+  classOf[DataSourceReader4xTSE],
+  classOf[DataSourceWriterTSE],
+  classOf[DataSourceWriterNeo4j4xTSE]
```

Comment:
Can we make naming consistent? `DataSourceReaderNeo4j4xTSE`

Reply:
done